### PR TITLE
Change pid file location for trusty

### PR DESCRIPTION
**Pull request by cdonati**
The os check was needed due to this issue: https://github.com/miketheman/nginx/issues/248

### CODE REVIEW

File: AppController/lib/nginx.rb [hunk 1]
```diff
@@ -769,12 +769,22 @@ def self.create_app_config(my_public_ip, my_private_ip, proxy_port,
 
   # Set up the folder structure and creates the configuration files necessary for nginx
   def self.initialize_config
+    nginx_pid_file = '/var/run/nginx.pid'
+    begin
+      os_version = `lsb_release -c`
+      if os_version.split.last == 'trusty'
+        nginx_pid_file = '/run/nginx.pid'
+      end
```
Comment (obino) [inline AppController/lib/nginx.rb]: Given that all modern distro have /run, and /var/run is a symlink to it, I would propose either to try /run also on precise, or to reverse the logic, and put the pid in /var/run only if it is precise, so we will be able to remove it as soon as we move away from precise.

**Comment by obino** [comment]
:+1: good find on the nginx issue!

**Comment by cdonati** [comment]
Precise: https://ocd.appscale.com:8080/job/Daily%20Build/936/
Trusty: https://ocd.appscale.com:8080/job/Daily%20Build/937/

**Comment by cdonati** [comment]
Precise: https://ocd.appscale.com:8080/job/Daily%20Build/940/
Trusty: https://ocd.appscale.com:8080/job/Daily%20Build/941/

**Pull request merged**